src/surfing_penguin/models.py

```diff
@@ -1,6 +1,36 @@
 """ models.py: Classes that tell the content in the database."""
 
-from surfing_penguin import db
+from surfing_penguin import db, api
+from flask_restplus import fields
+
+
+question = api.model("question_model", {
+        'content': fields.String,
+        'id': fields.Integer
+    })
+
```

Comment:
Why do you put the API model here? It should be in routes.py or other files. models.py is for the database. 
Also, you should add description for the API model.

Reply:
I think the Qstnr class will be recorded into the database, and it needs the api.model.
Or it should not work this way?